plugins/test/TranslationServer.js

```diff
@@ -322,7 +336,86 @@ describe('TranslationServer', function () {
             });
         });
 
-        it('should handle osmtotds POST of facility area feature', function() {
+        it('should handle OSM to GGDMv30 English POST of road line feature with width', function() {
+            //http://localhost:8094/osmtotds
+            var osm2trans = server.handleInputs({
+                osm: '<osm version="0.6" upload="true" generator="hootenanny"><way id="-8" version="0"><nd ref="-21"/><nd ref="-24"/><nd ref="-27"/><tag k="highway" v="road"/><tag k="uuid" v="{8cd72087-a7a2-43a9-8dfb-7836f2ffea13}"/><tag k="width" v="20"/><tag k="lanes" v="2"/></way></osm>',
+                method: 'POST',
+                translation: 'GGDMv30',
+                path: '/osmtotds'
+            });
```

Comment:
The full default tags get output with commit 29476ea24ce37b080aab2cf1f3920800ba24ac49, before you most recent changes.